input/tcplistener/tcplinelistener.go

```diff
@@ -63,9 +63,6 @@ func NewTCPLineListener(parentLogger logger.Logger, address string, testRecord f
 	taskCounter := &sync.WaitGroup{}
 	taskCounter.Add(1)
 
-	// call receiver.Destroy after taskCounter instead of at end of run(), as some connections might still be closing by then.
```

Comment:
It was wrong to destroy the receiver / orchestrator here, since there can be multiple inputs (not used atm) to a single global receiver.